config/locales/api.en.yml

```diff
@@ -25,7 +62,16 @@ en:
           format: int
           desc: "Total number of existing records"
 
+      request: |-
+        GET /api/v1/plant_lines?page=2&per_page=100 HTTP/1.1
+        Content-Type: application/json
+        Accept: application/json
+        X-BIP-Api-Key: ba6d4625927ede0db8768782994d9276cfff9ba79e8df8c831aedc453e850836
```

Comment:
Do you think it is confusing that the key shown here is not the real one used in sample api calls? I didn't want to expose user's key here so that user does not expose it by mistake (e.g. copy and paste somewhere).